pages/apis/webhooks/job_events.md.erb

```diff
@@ -8,15 +8,15 @@
 <tbody>
   <tr>
     <th><code>job.started</code></th>
-    <td><%= webhook_description "job.started" %></td>
+    <td>A command step has been started</td>
```

Comment:
What about "A command step job has been started"? You can't start a step, per-se. But jobs have "steps" which are like templates. Trick!

Reply:
Also, perhaps its "A command step job has started running on an agent"? I just checked the code, and that seems to be the case.

Reply:
Hah I had 'job' in my first draft, then took it out 😄 Happy to add it back in!

Reply:
Hah, cool! Yeah, I reckon let’s add it.